codec-http/src/main/java/io/netty/handler/codec/http/ClientCookieEncoder.java

```diff
@@ -119,6 +119,15 @@ private static void encode(StringBuilder buf, Cookie c) {
         // rawValue > value > ""
         String value = c.rawValue() != null ? c.rawValue()
                 : c.value() != null ? c.value() : "";
+
+        if (ServerCookieDecoder.containsInvalidCookieNameOctet(c.name(), 0, c.name().length())) {
```

Comment:
There's already a CookieEncoderUtil but it would be weird as well to use it in for decoding. I guess it could be renamed but what about compat? Is it ok as 4.1 is still in beta?